lib/polist/service.rb

```diff
@@ -52,6 +71,7 @@ def initialize(params = {})
     def call; end
 
     def run
+      call_middlewares
```

Comment:
This way the middlewares are only going to be called if I do `Example.run` and not if I do `Example.call`. And the only difference between these 2 methods should be that the latter raises exception in case of failure.